src/status_im/contexts/settings/wallet/network_settings/view.cljs

```diff
@@ -0,0 +1,108 @@
+    :blur?     true
+    :list-type :settings}])
+
+(defn layer-2-settings
+  [networks-by-name]
+  [quo/category
+   {:key       :layer-2-settings
+    :label     (i18n/label :t/layer-2)
+    :data      (map make-network-settings-item
+                    [(:optimism networks-by-name)
+                     (:arbitrum networks-by-name)])
+    :blur?     true
+    :list-type :settings}])
+
+(defn testnet-mode-setting
+  [{:keys [on-enable on-disable]}]
+  (let [testnet-mode-enabled? (rf/sub [:profile/test-networks-enabled?])
+        on-change-testnet     (rn/use-callback
+                               (fn [active?]
+                                 (if active? (on-enable) (on-disable)))
+                               [on-enable on-disable])]
+    {:blur?        true
+     :title        (i18n/label :t/testnet-mode)
+     :action       :selector
+     :image        :icon
+     :image-props  :i/settings
+     :action-props {:on-change on-change-testnet
+                    :checked?  (boolean testnet-mode-enabled?)}}))
+
+(defn advanced-settings
+  [{:keys [enable-testnet disable-testnet]}]
+  [quo/category
+   {:key       :advanced-settings
+    :label     (i18n/label :t/advanced)
+    :data      [(testnet-mode-setting {:on-enable enable-testnet
+                                       :on-disable disable-testnet})]
+    :blur?     true
+    :list-type :settings}])
+
+(defn on-change-testnet
+  [{:keys [enable? theme]}]
+  (rf/dispatch [:show-bottom-sheet
+                {:content (fn [] [testnet/view {:enable? enable?}])
+                 :theme   theme}]))
+
+(defn view
+  []
+  (let [insets           (safe-area/get-insets)
+        theme            (quo.theme/use-theme)
+        networks-by-name (rf/sub [:wallet/network-details-by-network-name])
+        enable-testnet   (rn/use-callback
+                          (fn []
+                            (on-change-testnet {:theme theme
+                                                :enable? true}))
+                          [theme])
+        disable-testnet   (rn/use-callback
+                           (fn []
+                             (on-change-testnet {:theme theme
+                                                 :enable? false}))
+                           [theme])]
+    [quo/overlay
+     {:type            :shell
+      :container-style (style/page-wrapper (:top insets))}
+     [quo/page-nav
+      {:key        :header
+       :background :blur
+       :icon-name  :i/arrow-left
+       :on-press   navigate-back}]
+     [rn/view {:style style/title-container}
+      [quo/standard-title
+       {:title               (i18n/label :t/network-settings)
+        :accessibility-label :network-settings-header}]]
+     [rn/view {:style (style/settings-container (:bottom insets))}
+      (when networks-by-name
+        [rn/view {:style style/networks-container}
+         [mainnet-settings (:mainnet networks-by-name)]
+         [layer-2-settings networks-by-name]])
```

Comment:
Here we need to make sure we have networks to display before attempting to display them. This seems important for when we logout the user account, because it seems we may be clearing the wallet networks data from the app state.